drivers/timer/cortex_m_systick.c

```diff
@@ -115,9 +115,8 @@ static uint32_t elapsed(void)
 }
 
 /* Callout out of platform assembly, not hooked via IRQ_CONNECT... */
-void sys_clock_isr(void *arg)
+ISR_DIRECT_DECLARE(sys_clock_isr)
```

Comment:
I don't know when indirect->direct changing was really happened, but actual "sys_clock_isr" is a direct isr call. And yes, I was sure that ISR_DIRECT_DECLARE does this: "They must be implemented in such a way that it is safe to put them directly in the vector table.  For ISRs written in C, The ISR_DIRECT_DECLARE() macro will do this automatically.", but actually it does not. Will fix my implementation.

BTW IMHO the current driver should also be marked "broken" because it still uses the normal C calling convention.